test/harmony-regression/HarmonyRegression.ipynb

```diff
@@ -61,7 +61,10 @@
    "source": [
     "coverages_root = '{root}/{collection}/ogc-api-coverages/1.0.0/collections/{variable}/coverage/rangeset'\n",
     "\n",
-    "is_not_prod = harmony_host_url == 'https://harmony.uat.earthdata.nasa.gov' or harmony_host_url == 'https://harmony.sit.earthdata.nasa.gov' or re.search(\"http:\\/\\/internal-harmony-.*us-west-2\\.elb.amazonaws\\.com\", harmony_host_url)\n",
+    "if harmony_host_url == 'https://harmony.earthdata.nasa.gov':\n",
```

Comment:
The changes on this file look a little extensive - they aren't. There's mainly some formatting clean-up.

The main changes are the addition of the `harmony-py` code, and then the last test for the Harmony/downloader service.

In a future PR, I'd like to clean-up this notebook a little bit using `harmony-py` more extensively, instead of `reqeusts` as it currently is. But that's out of scope here.